README.md

```diff
@@ -165,6 +165,7 @@ type, the labels used to slice the metric, and a description.
 | `sth_timestamp` | GaugeVec | `uri` | Timestamp of fetched STH |
 | `sth_age`       | GaugeVec | `uri`            | Elapsed time since timestamp of fetched STH |
 | `sth_failures`  | CounterVec | `uri` | Count of failures fetching a STH |
+| `sth_fetch_total`  | CounterVec | `uri` | Count total number of get-sth calls made against each observed CT log |
```

Comment:
Thanks for updating the README!

I think you used a copy of the Prometheus stat description from before you adjusted it in the code:

```suggestion
| `sth_fetch_total`  | CounterVec | `uri` | Count of total number of get-sth calls made against each monitored CT log |
```

Reply:
Yup, I totally did.